.github/commitlint.config.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable import/no-extraneous-dependencies */
-const maxLineLength = 72
+const { maxLineLength } = require('@commitlint/ensure')
```

Comment:
This looks like a revert on one of the other two patches.

Reply:
yes, I was trying to fix that but node-js has defeated me, I've created a separate issue #633  and I'll clean this PR in a second